examples/apps/tree-comparison/src/model/schema.ts

```diff
@@ -0,0 +1,33 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { leaf, SchemaBuilder, TypedField, TypedNode } from "@fluid-experimental/tree2";
+
+// By importing the leaf library we don't have to define our own string and number types.
+const builder = new SchemaBuilder({ scope: "inventory app", libraries: [leaf.library] });
+
+const inventoryItem = builder.struct("Contoso:InventoryItem-1.0.0", {
+	// REV: I added an ID here because I didn't find a unique identifier on the node.
```

Comment:
unlike tree, tree2's editing system is not built on explicitly stamped ids on the nodes. This doesn't duplicate anything afaik. Including it in the schema as 'real life inventory items have ids' seems fine (though maybe write a note that that's the intent)

Reply:
In March/April, there will actually be UUID identifiers for nodes, so this TODO will be true then :P

Reply:
Yep I removed this REV comment in the latest set of changes (in `main` now) for the "real life inventory items have ids" reason.

Even if UUID identifiers are added to nodes in the future, I'd still probably recommend the usage principle I mention in my other comment for legacy ST: "The NodeId is not part of your schema so don't act like it is".